exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -73,6 +73,10 @@ func (e *exporter) Start(ctx context.Context, host component.Host) error {
 			return
```

Comment:
In most of CW Logs use case, the log group/stream needs to be created in flight for the first time use. I think we should create Stream based on Config instead of failing the exporter.